k8s/namespaces/lau/lau-case-backend/lau-case-backend.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: lau-case-backend
+spec:
+  releaseName: lau-case-backend
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/lau-case-backend
+  values:
+    java:
+      memoryRequests: "1024Mi"
```

Comment:
have you tested that this is necessary?

overriding defaults should only be done if proven via performance testing

Reply:
I've removed these overrides. Copied what we did with PCQ as the apps are of similar size.